lib/features/di-ordering/BpmnDiOrdering.js

```diff
@@ -0,0 +1,39 @@
+// import { getDi } from '../../draw/BpmnRenderUtil';
+// import { getBusinessObject } from '../../util/ModelUtil';
+//
+// import {
+//   filter,
+//   map
+// } from 'min-dash';
+//
+// import { selfAndAllChildren } from 'diagram-js/lib/util/Elements';
+
+
+var HIGH_PRIORITY = 2000;
+
+export default function BpmnDiOrdering(eventBus, canvas) {
+
+  eventBus.on('saveXML.start', HIGH_PRIORITY, orderDi);
+
+  function orderDi() {
+    // we do not order, as ordering does not work with the participant one to many mapping.
+
+    // var root = canvas.getRootElement(),
+    //     rootDi = getBusinessObject(root).di,
+    //     elements,
+    //     diElements;
+    //
+    // elements = selfAndAllChildren([ root ], false);
+    //
+    // // only bpmndi:Shape and bpmndi:Edge can be direct children of bpmndi:Plane
+    // elements = filter(elements, function(element) {
+    //   return element !== root && !element.labelTarget;
+    // });
+    //
+    // diElements = map(elements, getDi);
+    //
+    // rootDi.set('planeElement', diElements);
```

Comment:
Maybe just add an explanation on what exactly is being done here? Took me a second to see we are overwriting this so it does nothing. Get rid of the code and just explain I would say.

Reply:
Yeah, the comment about that is hidden in the comment and not quite clear.